changelog.txt

```diff
@@ -1,3 +1,11 @@
+Version 1.21.0
+=============
+- Add support for claims and client capabilities in Service Fabric scenarios (#929, #943)
```

Comment:
Maybe mention "token revocation" in some way?

Reply:
Just changed it to "Add support for claims, client capabilities, and token revocation in Service Fabric scenarios"